transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannelConfig.java

```diff
@@ -58,11 +58,6 @@
 
         return true;
     }
-    @Override
-    public EpollDomainSocketChannelConfig setMaxMessagesPerRead(int maxMessagesPerRead) {
```

Comment:
@normanmaurer Yeah, we can't just remove it. How about deprecating it and delegate to `Max*Allocator` if the current recv allocator is of compatible type, or throw an exception if it's incompatible?